native/cocos/renderer/pipeline/custom/LayoutGraphGraphs.h

```diff
@@ -1271,7 +1271,7 @@ addVertex(Component0&& c0, Component1&& c1, ValueT&& val, LayoutGraph& g, Layout
 
 template <class Tuple>
 void addVertexImpl(RenderStageTag /*tag*/, Tuple &&val, LayoutGraph &g, LayoutGraph::Vertex &vert) {
-    invoke_hpp::apply(
+    std::apply(
```

Comment:
use c++17 std::apply. remove previous c++14 implementation.

Reply:
Should test on iOS 11. Not sure if it works.

Reply:
Sure. Generally speaking, std::apply is pure template, so it does not depend on any runtime feature.